src/components/country/country.css

```diff
@@ -0,0 +1,80 @@
+.country-container {
+  width: 100%;
+  height: 100%;
+  margin-top: 5rem;
+  padding: 0 5rem;
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
+
+.country-card {
+  max-width: 20%;
+  background-color: var(--color-white);
+  margin-right: 4rem;
+  margin-bottom: 8rem;
+  cursor: pointer;
+  text-decoration: none;
+  box-shadow: 5px 5px 10px 0 #dfd5d5;
+  color: var(--light-mode-text);
+}
+
+.country-image {
+  display: block;
+  width: 100%;
+  height: 50%;
+}
+
+.country-content {
+  padding: 2rem;
+}
+
+.country-content p {
+  margin-top: 1rem;
+  font-weight: 600;
+}
+
+.country-content p span {
+  font-weight: 300;
+}
+
+.countryheader {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+}
```

Comment:
- [ ] how about we move this country name and the arrow to the top and make sure the arrow is on top right corner to match the original design layout
currently 
![image](https://user-images.githubusercontent.com/97521557/233278104-9b6341d5-d176-4cf6-bcb1-5b4e38aaf2c1.png)

design 
![image](https://user-images.githubusercontent.com/97521557/233278251-383bb938-c692-4fff-a53e-519aba01cec0.png)

notice the arrow is on top right :+1: